Sources/MongoSwift/MongoConnectionString.swift

```diff
@@ -467,12 +467,10 @@ public struct MongoConnectionString: Codable, LosslessStringConvertible {
         // Parse the defaultAuthDB if present.
         if userHostsAndAuthDB.count == 2 && !userHostsAndAuthDB[1].isEmpty {
             let defaultAuthDB = try userHostsAndAuthDB[1].getPercentDecoded(forKey: "defaultAuthDB")
-            for character in Self.forbiddenDBCharacters {
-                if defaultAuthDB.contains(character) {
-                    throw MongoError.InvalidArgumentError(
-                        message: "defaultAuthDB contains invalid character: \(character)"
-                    )
-                }
+            for character in Self.forbiddenDBCharacters where defaultAuthDB.contains(character) {
```

Comment:
this and the other change in the file were linter changes due to the version updates. there's a swiftlint rule around preferring "for... where" over a "for" loop with a single nested "if", and I guess swiftlint previously missed these violations